examples/bodymovin-animation.amp.html

```diff
@@ -30,10 +31,15 @@ <h2>Example of an animation that doesn't autoplay</h2>
   <amp-bodymovin-animation id="anim" layout="fixed" width="200" height="200" src="https://nainar.github.io/loader.json" loop="true" name="ripple_1" noautoplay>
   </amp-bodymovin-animation>
 
+  <h2>Example of a bodymovin animation controlled by scroll position</h2>
+  <amp-bodymovin-animation layout="fixed" width="200" height="200" src="https://nainar.github.io/loader.json" loop="true" id="bodymovin-scrollable">
+  </amp-bodymovin-animation>
+  <amp-position-observer intersection-ratios="0" on="scroll:bodymovin-scrollable.seekTo(percent=event.percent)" layout="nodisplay">
```

Comment:
intersection-rations needs to be 1 so:

```
  <amp-position-observer intersection-ratios="1" on="scroll:scrollable.seekTo(percent=event.percent)" layout="nodisplay">
  </amp-position-observer>
```

Reply:
I changed it to `1` :) (Note that [any value between `0` and `1` is valid](https://github.com/ampproject/amphtml/blob/master/extensions/amp-position-observer/amp-position-observer.md#intersection-ratios-optional) here, by the way.)